Bug fixes and small alterations (#115)
rostools/r-cubed-advanced: sessions/smoother-collaboration.qmd

        packages depend on, are installed within your project,
        installation times can sometimes be a bit long.

>     -   When you restore a library from a renv.lock file that utilize

Reviewer: ```suggestion
    -   When you restore a library from a `renv.lock` file that uses
```

>         old package versions, it can sometimes be cumersome to

Reviewer: ```suggestion
        old package versions, it can sometimes be difficult to
```

However, in science, we want to be more explicit rather than implicit.
The way `{renv}` explicitly adds to the `renv.lock` file is by only

> scanning the `DESCRIPTION` file. While we allready selected to use

Reviewer: ```suggestion
scanning the `DESCRIPTION` file. While we have already selected to use
```

explicit declaration when initilizing `{renv}`, this option is specific

> for our R instance. In order for `{renv}` to always work explicit, we

Reviewer: ```suggestion
to our R instance. In order for `{renv}` to always work explicitly, we
```

> need to set an project option for it. This option needs to be added to

Reviewer: ```suggestion
need to set a project option for it. This option needs to be added to
```
